lightly/utils/benchmarking/metric_callback.py

```diff
@@ -0,0 +1,30 @@
+from collections import defaultdict
+from typing import List
+
+from pytorch_lightning import LightningModule, Trainer
+from pytorch_lightning.callbacks import Callback
+from torch import Tensor
+
+
+class MetricCallback(Callback):
```

Comment:
This is needed to get max performance over all epochs. PyTorch Lightning only stores logs from last epoch. This callback stores logs at the end of every epoch.